src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -119,7 +124,17 @@ function getCSharpInputTypeKindByIntrinsicModelName(
         case "float64":
             return InputTypeKind.Float64;
         case "string":
-            return InputTypeKind.String;
+            switch (format?.toLowerCase()) {
```

Comment:
Does format only applies to string?

Reply:
Yes. @format is only applied on string, its string format.